src/api.v2/helpers/worker/generateETag.js

```diff
@@ -0,0 +1,36 @@
+const hash = require('object-hash');
+const config = require('../../../config');
+const getExperimentBackendStatus = require('../backendStatus/getExperimentBackendStatus');
+
+const createObjectHash = (object) => hash.MD5(object);
+
+const generateETag = async (
+  experimentId,
+  body,
+  extraDependencies,
+  extras,
+  cacheUniquenessKey = null,
+) => {
+  const backendStatus = await getExperimentBackendStatus(experimentId);
+  const { pipeline: { startDate: qcPipelineStartDate } } = backendStatus;
```

Comment:
is it ok to use the qc start date for all the different pipelines, like `subset`,  cell-level `metadata`, `rds upload` etc...? @cosa65 @gerbeldo